framework/resource/metricsresource/resource.go

```diff
@@ -131,41 +151,56 @@ func (r *Resource) Name() string {
 }
 
 func (r *Resource) ApplyCreateChange(ctx context.Context, obj, createState interface{}) error {
-	o := "ApplyCreatePatch"
+	sl := r.name
+	rl := r.resource.Underlying().Name()
+	ol := "ApplyCreatePatch"
```

Comment:
Nitpick: I think you can just create a var `operationErrorCounter.WithLabelValues` here instead of having this criptic vars here.

Reply:
I don't follow. What do you mean?

Reply:
I think you can assign `something := operationErrorCounter.WithLabelValues(r.name, r.resource.Underlying().Name(), "ApplyCreatePatch")` and then do `something.Inc()`.

Reply:
Why would this be better?

Reply:
I thought it's a good thing you don't have to call `WithLabelValues` multiple times with the same values.

Reply:
We call three different metric valriables, so this does not help.